### PR TITLE
Show all search new

### PR DESCRIPTION
**Pull request by rusetskiy1974**

### CODE REVIEW

File: user_assistant/commands.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 FIND_CONTACT = 'find_contact'
 SHOW_BIRTHDAY = 'show_birthday'
 SORT_FILES = 'sort_files'
+SHOW_ALL = 'show_all'
```
Comment (AlekseyP18) [inline user_assistant/commands.py]: show_all_contacts

File: user_assistant/handlers/address_book/show_all.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from user_assistant.console.console import Console
+
+def show_all(book):
```
Comment (AlekseyP18) [inline user_assistant/handlers/address_book/show_all.py]: show_all_contacts

і файл відповідно

File: user_assistant/handlers/address_book/search_contact.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from user_assistant.console.console import Console
+
+def search_contact(book):
+    while True:
+        string = Console.input(f'Input keyword for name or phone :')
```
Comment (AlekseyP18) [inline user_assistant/handlers/address_book/search_contact.py]: Console.input('Input contact name or phone: ')

File: user_assistant/handlers/address_book/search_contact.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        if string:
+            break
+        Console.print_error('Value is missing, try agane')
```
Comment (AlekseyP18) [inline user_assistant/handlers/address_book/search_contact.py:8]: 'Value is missing, try again'

File: user_assistant/handlers/address_book/search_contact.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+    result = book.search(string)
+
+    if result:
+        for record in result:
+            Console.print_success(record)
+         
+    else:
+        Console.print_error(f'There is no any contact with keyword: {string}')
```
Comment (AlekseyP18) [inline user_assistant/handlers/address_book/search_contact.py]: Console.print_error(f'There is no any contact by your request')

**Review by AlekseyP18 — APPROVED** [review]

**Review by PotapovALeksey — APPROVED** [review]

**Pull request merged**